platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/IdpEndpoint.java

```diff
@@ -382,7 +380,7 @@ private Response getErrorResponse(String relayState, AuthnRequest authnRequest,
             throws WSSecurityException, IOException, SimpleSign.SignatureException {
         LOGGER.debug("Creating SAML Response for error condition.");
         org.opensaml.saml.saml2.core.Response samlResponse = SamlProtocol.createResponse(
-                SamlProtocol.createIssuer(systemBaseUrl.constructUrl("/idp/login", true)),
+                SamlProtocol.createIssuer(SystemBaseUrl.constructUrl("/idp/login", true)),
```

Comment:
I noticed the pom isn't embedding common-system. Won't this fail to find SystemBaseUrl at runtime?